src/data/skill-desc.csv

```diff
@@ -393,5 +393,5 @@
 "レース終盤に前方が詰まると\nお告げ次第で道をわずかに切り開く","<size=22>Slightly easier to find an opening if blocked at the front in Final Leg</size>"
 "レース終盤で3番手の時に負けそうに\nなると速度がわずかに上がる","<size=22>When you're in 3rd place at the end of the race and about to lose, your speed slightly increases</size>"
 "レースを冷静に運ぶことができると残り\n200mで実力発揮、速度が少し上がる","<size=22>After proceeding through the race calmly, your speed increases a little during the last 200m</size>"
-"最終直線で前の方で追いすがっていると\n速度がわずかに上がる",""
-"最終コーナー以降に前めで力が尽きかけ\n詰め寄られると速度がわずかに上がる",""
+"最終直線で前の方で追いすがっていると\n速度がわずかに上がる","<size=22>Slightly increase speed in the last straight when chasing someone close to the lead</size>"
+"最終コーナー以降に前めで力が尽きかけ\n詰め寄られると速度がわずかに上がる","<size=22>Slightly increase speed after the last corner when stamina runs out being one of the lead</size>"
```

Comment:
Does this fit? lol  
I can check later ingame

Reply:
the 3 and higher star skill variants fit just fine 
https://files.catbox.moe/32h48s.PNG
these two are lower star skill variants, i doubt they even show in game but should be fine if they do